### DISCUSSION TITLE
Callbacks to the `close` and `open` exposed methods of BDropdown

### DISCUSSION THREAD

**Discussion by kbamponsem** [cdmoro/bootstrap-vue-3#1514] [Ideas]
I have this use-case where I would want to fire up some events after the dropdown is closed.

Someone could correct me and point me the right direction if this has already been implemented. 

The idea would be that we submit callback functions (openCallback, closeCallback) that will be executed when the dropdown is closed.

Any help would be greatly appreciated.

Thanks.

**Comment by VividLemon** [comment]
https://github.com/bootstrap-vue-next/bootstrap-vue-next/blob/69e6fe01dbe3e8f1605000966594a43e9143ea05/packages/bootstrap-vue-next/src/components/BDropdown/BDropdown.vue#L122
https://github.com/bootstrap-vue-next/bootstrap-vue-next/blob/69e6fe01dbe3e8f1605000966594a43e9143ea05/packages/bootstrap-vue-next/src/components/BDropdown/BDropdown.vue#L120

**Reply by kbamponsem** [reply]
Thanks @VividLemon
